scripts/dracula.sh

```diff
@@ -298,9 +298,9 @@ main()
       continue
     fi
 
-    if [ $plugin = "rpi-temp" ]; then
+    if [ $plugin = "sys-temp" ]; then
       IFS=' ' read -r -a colors <<< $(get_tmux_option "@dracula-rpi-temp-colors" "green dark_gray")
```

Comment:
Does it have to change as well?

Reply:
Yes. It should in order to keep things consistent.

Reply:
Agreed, it should be `dracula-sys-temp-colors`.
@ethancedwards8  Do we have a place where all the parameters are documented? I am struggling to keep up with the latest and greatest in my `tmux.conf` file.

Reply:
Addressed on https://github.com/dracula/tmux/pull/307/commits/8efdc955df0d6dabca42d65c9bd5fe69a2651379. Thank you!

Reply:
> Agreed, it should be `dracula-sys-temp-colors`. @ethancedwards8 Do we have a place where all the parameters are documented? I am struggling to keep up with the latest and greatest in my `tmux.conf` file.

Kinda. INSTALL.md. It isn't 100% complete though. That is something we are working on.